.codecov.yml

```diff
@@ -1,5 +1,6 @@
 ---
 codecov:
+  comment: false
```

Comment:
it deactivates the codecov commenting hopefully, @mr-karan did that for coala-bears at least and I don't see comments there, maybe it'll silence codecov finally